docs/src/getting-started/wallets.md

```diff
@@ -118,8 +118,25 @@ let password = "my_master_password";
 let uuid = wallet.encrypt(&dir, password).unwrap();
 ```
 
+## Checking balances and coins
+
+First, one should keep in mind that, with UTXOs, each _coin_ is unique, and corresponds to a unique UTXO. So, when you want to query the balance for a given asset ID, you want to query the sum of the amount in each unspent coin. This is done very easily with a wallet:
```

Comment:
Since we're talking about the uniqueness of UTXOs, we should also mention, after `each _coin_ is unique, and corresponds to a unique UTXO`, that a coin also holds an amount, maybe use the USD bills analogy: a 10 dollars bill is a single "coin" with the amount `10`, a 50 dollars bill is a single "coin" wit the amount `50`, and, likewise, you can have a single coin here with an arbitrary amount.

The concept of coin and amount is often confusing for newcomers!